guides/common/modules/proc_enabling-pull-mode-on-proxy.adoc

```diff
@@ -0,0 +1,25 @@
+[id="enabling-pull-mode-on-proxy_{context}"]
+= Enabling Pull Mode on {SmartProxy}
+
+Use the following procedure to enable pull-based transport on {SmartProxy}.
+
+.Procedure
+. Connect to {SmartProxyServer} using SSH.
+. Enable the pull mode:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --foreman-proxy-plugin-remote-execution-script-mode=pull-mqtt
+----
++
+When the installer command succeeds, the MQTT broker listens on port 1883.
```

Comment:
```suggestion
```

IMHO not necessary because a user will see this anyway.

Reply:
I'd prefer to keep that sentence for "verification".